### ISSUE TITLE
playwright._impl._errors.TimeoutError: Timeout 30000ms exceeded.

### ISSUE THREAD

**Issue by tj22777** [evmer/scribd-downloader#22]
evmer/scribd-downloader/run.py", line 58, in <module>

    font_style = page.locator('#fontfaces').inner_html()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/playwright/sync_api/_generated.py", line 16146, in inner_html
    self._sync(self._impl_obj.inner_html(timeout=timeout))
  File "/usr/lib/python3/dist-packages/playwright/_impl/_sync_base.py", line 113, in _sync
    return task.result()
           ^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/playwright/_impl/_locator.py", line 429, in inner_html
    return await self._frame.inner_html(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/playwright/_impl/_frame.py", line 623, in inner_html
    return await self._channel.send("innerHTML", locals_to_params(locals()))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/playwright/_impl/_connection.py", line 59, in send
    return await self._connection.wrap_api_call(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/playwright/_impl/_connection.py", line 509, in wrap_api_call
    return await cb()
           ^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/playwright/_impl/_connection.py", line 97, in inner_send
    result = next(iter(done)).result()
             ^^^^^^^^^^^^^^^^^^^^^^^^^
playwright._impl._errors.TimeoutError: Timeout 30000ms exceeded.

**Comment by Oxeda** [comment]
did you solve it yet my friend?

**Comment by tj22777** [comment]
nope, forget this one, use another repo is working, this one is not working

**Comment by Oxeda** [comment]
> nope, forget this one, use another repo is working, this one is not working

Got any suggestions? Ty

**Comment by krupuk1** [comment]
> nope, forget this one, use another repo is working, this one is not working

Could you share the link to the working repository?

**Comment by tj22777** [comment]
Try this, read the instruction
 https://github.com/rkwyu/scribd-dl

**Comment by krupuk1** [comment]
> Try this, read the instruction https://github.com/rkwyu/scribd-dl

This doesn't work for downloading ebooks from Everand

**Comment by tj22777** [comment]
which Everand? it's stll working for me.
